README.md

```diff
@@ -45,8 +45,8 @@ more flexibility.
 
 ## Installation
 
-```bash
-curl -sSL https://raw.githubusercontent.com/soywod/himalaya/master/install.sh | bash
+```sh
+sh -c "$(curl -sSL https://github.com/soywod/himalaya/raw/master/install.sh)"
```

Comment:
Why replacing `raw.githubusercontent.com` by `github.com`? In fact the cURL command can be simplified by `curl -sS https://raw.githubusercontent.com/soywod/himalaya/master/install.sh`, it prevents a redirection.